mypy_django_plugin/transformers/models.py

```diff
@@ -440,7 +440,7 @@ def run_with_model_cls(self, model_cls: Type[Model]) -> None:
         self.add_new_node_to_model_class("_default_manager", default_manager, is_classvar=True)
 
 
-class AddRelatedManagers(ModelClassInitializer):
+class AddReverseLookups(ModelClassInitializer):
```

Comment:
It took me quite some time to figure out that the reverse one to one was set by the plugin in this initializer.

I'm all up for an even better name that describes that this handles the reverse parts of relationships..